app.js

```diff
@@ -24,5 +25,6 @@ app.registerStore(RouteStore);
 app.registerStore(ApplicationStore);
 app.registerStore(PropertyListingsStore);
 app.registerStore(LocationStore);
+app.registerStore(GeolocationStore);
```

Comment:
Do we need a separate store for geolocation? LocationStore doesn't suffice?

Reply:
i think so because there is a difference between them, makes it clear what you're trying to access